Makefile

```diff
@@ -516,4 +516,4 @@ dev-release:
 ## validate the operator by installing the releases
 validate-release: setup-venv
 	$(Q)$(PYTHON_VENV_DIR)/bin/pip install yq==2.10.0
-	BUNDLE_VERSION=$(BASE_BUNDLE_VERSION) CHANNEL="alpha" ./hack/validate-release.sh
+	BUNDLE_VERSION=$(BASE_BUNDLE_VERSION) CHANNEL="alpha-0.2" ./hack/validate-release.sh
```

Comment:
let's call the channel `beta`.

Reply:
Are we going to increment API version also into `beta`? cc. @akashshinde

Reply:
I don't think we are in a state to provide beta level compatibility? 🤔 maybe in a future version ( also a future tech preview version )

Reply:
+1 - channel name and API stability level are two orthogonal things, IMHO.

Reply:
So, I can change the channel name as `beta`. Is that correct understanding?